src/main/java/com/whiteclarkegroup/liquibaselinter/config/ConfigLoader.java

```diff
@@ -5,20 +5,25 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.List;
+import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Stream;
+
+import static java.lang.System.getProperty;
+import static java.util.stream.Collectors.toList;
 
 public class ConfigLoader {
 
     public static final String LQLINT_CONFIG = "/lqlint.json";
-    public static final String LQLLINT_CONFIG = "/lqllint.json";
```

Comment:
Good that we are dropping this - was it mentioned in the docs anywhere?

Reply:
Nope it was not documented anywhere